files/rubric.txt

```diff
@@ -0,0 +1,250 @@
+teaching, how Software and Data Carpentry can help in your community.
+Spaces are limited, so use this form to share with us what excites you
+about spreading our teaching to your organization. We will notify
+applicants as limited spaces become available. If you have any
+questions, please mail <training@software-carpentry.org>.
+
+If you would like to accelerate the process, check out our
+[partnership program](http://software-carpentry.org/scf/join/).
+Organizational partners make ongoing commitments to supporting our
+organization and are prioritized for instructor training. If you need
+help making the case at your organization, feel free to contact us at
+<partnerships@software-carpentry.org>, we'd be happy to help.
+
+Please note that as a condition of taking this training:
+
+-   You are required to abide by our code of conduct, which can be found
+    at <http://software-carpentry.org/conduct/> and
+    <http://datacarpentry.org/code-of-conduct/>.
+-   You must complete three short tasks after the course in order to
+    complete certification. The tasks are described at
+    <http://swcarpentry.github.io/instructor-training/checkout/>, and
+    take a total of approximately 8-10 hours.
+-   You are expected to teach at a Software Carpentry or Data Carpentry
+    workshop within 12 months of the course.
+
+For more information on Software and Data Carpentry instructor training,
+please see <http://swcarpentry.github.io/instructor-training/>.
+
+> Applications for the open Fall 2016 online instructor training class are
+> now closed, but we are still accepting applications for subsequent
+> classes to be scheduled.
+>
+> If you are scheduled to receive training at a partner site, and have
+> been asked to fill in this form to give us information about yourself,
+> please do so and we will follow up when your site's training has been
+> scheduled.
+
+
+
+01. Group name: [____________________]
+  If you are applying at the same time as friends or colleagues, pick
+  a name for your group (e.g., the name of your institution) and fill
+  that in, and we will try to put you in the same training class.
+
+02. Personal (given) name: [____________________]
+
+03. Family name (surname): [____________________]
+
+04. Email address: [____________________]
+
+05. GitHub username: [____________________]
+
+
+
+06. What is your current occupation/career stage?
+  Please choose the one that best describes you.
+
+[_] Prefer not to say
+[_] Undergraduate student
+[_] Graduate student
+[_] Post-doctoral researcher
+[_] Faculty
+[_] Research staff (including research programmer)
+[_] Support staff (including technical support)
+[_] Librarian/archivist
+[_] Commercial software developer
+[_] Other [____________________]
+
+
+
+07. Affiliation: [____________________]
+
+08. Location: [____________________]
+  Please give city, and province or state if applicable
+
+09. Country: [--pulldown--]
+
+10. [_] This a smaller, remote, or less affluent institution.
```

Comment:
Chances are people in the relevant areas may not know what the word 'affluent' means. Perhaps use 'a smaller or remote institution, or one with little funds'